test/integration/webhook/v1alpha2/clusterqueue_test.go

```diff
@@ -128,42 +142,70 @@ var _ = ginkgo.Describe("ClusterQueue Webhook", func() {
 			gomega.Expect(errors.IsForbidden(err)).Should(gomega.BeTrue(), "error: %v", err)
 		})
 
-		ginkgo.It("Should have non-negative quota value when creating", func() {
-			ginkgo.By("Creating a new clusterQueue")
-			cq := testing.MakeClusterQueue("cluster-queue").Resource(
-				testing.MakeResource("cpu").Flavor(testing.MakeFlavor("x86", "-1").Obj()).Obj(),
-			).Obj()
+		ginkgo.DescribeTable("ClusterQueue", func(cq *kueue.ClusterQueue, isForbidden, isInvalid bool) {
```

Comment:
Maybe also something like this?

```suggestion
		ginkgo.DescribeTable("ClusterQueue", func(cq *kueue.ClusterQueue, errorType string) {
```

and then:
```golang
switch errorType {
	case isForbidden:
```

Reply:
The logic and constants can even be part of `test/integration/framework`